public/scripts/controller/rsvp_controller.js

```diff
@@ -0,0 +1,55 @@
+define([], function() {
+  var rsvpController = function($scope, $http) {
+    $scope.title = "RSVP Y'all";
+    $scope.guest = {};
+    $scope.attendingOptions = [
+      {value: true, label: 'in person'},
+      {value: false, label: 'in spirit'}
+    ]
+
+    $scope.submitForm = function() {
+      var parsedResponse = parseRsvpFormResponse($scope.guest);
+
+      console.log('FIRE THE TORPEDOES!!!');
+
+      $http.post('/create_guest', {
+        guest: parsedResponse
+      })
+      .success(function() {
+        console.log('success!!!!');
+      })
+      .error(function() {
+        console.log('we fucked up');
+      });;
+    };
+
+    function parseRsvpFormResponse(response) {
+      return {
+        first_name: response.firstName,
+        last_name: response.lastName,
+        plus_ones: response.plus_ones,
+        phone: response.phone,
+        email: response.email,
+        attending: parseAttendingResponse(response)
+      }
+    };
+
+    function parseAttendingResponse(response) {
+      if (response.willAttend) {
```

Comment:
Couldn't you just remove that outer conditional and return 'neither' as the default, since all the other attending_ attributes will be undefined and fall through? Obviously not a big deal either way...

```
if (response.attendingSaratoga && response.attendingThailand) {
} else if (...) {
} else if (...) {
} else {
  return 'neither';
}
```